src/cpp/polars/Series.cpp

```diff
@@ -505,6 +505,33 @@ namespace polars {
         return m;
     }
 
+    bool Series::empty() const {
+        return (index().is_empty() & values().is_empty());
+    }
+
+    Series Series::head(int rows) const  {
+        Series ser = Series(values(), index());
+        if(rows >= ser.size()){
+            return ser;
+        } else {
+            arma::uvec indices = arma::conv_to<arma::uvec>::from(polars::numc::arange(0, rows));
+            return ser.iloc(indices);
```

Comment:
It would be nice to put the slicing logic into `.iloc(int start, int stop, int step=1)` so it can be called like `ser.iloc(0, -10)` to get all but the last 10 etc - then `.head(n)` would just wrap `.iloc(0, n)` and `.tail(n)` would wrap `.iloc(-n, -1)`

Reply:
Added todos